app/pages/lab-pages-editor/components/WorkflowHeader.jsx

```diff
@@ -10,13 +11,37 @@ import { useWorkflowContext } from '../context.js';
 import strings from '../strings.json';
 
 export default function WorkflowHeader({
-  projectId = ''
+  currentTab = 0,
+  projectId = '',
+  setCurrentTab = () => {},
+  tabs = []
 }) {
   const { workflow } = useWorkflowContext();
   const returnUrl = `/lab/${projectId}/workflows`;
 
-  function onClick() {
-    console.log('TODO');
+  // When clicking a tab button, make that tab active. This is pretty straightforward.
+  function onClick(e) {
+    const { tab } = e.target.dataset;
+    setCurrentTab(parseInt(tab));
```

Comment:
Setting the state of component A from component B isn't recommended in React. Maybe have an `onChange` callback that runs when the state of `WorkflowHeader` changes?